py3status/modules/weather_owm.py

```diff
@@ -0,0 +1,587 @@
+# -*- coding: utf-8 -*-
+'''
+Ultimately customizable weather module based on the IP-API Geolocation API
+(http://ip-api.com) and the OpenWeatherMap API (https://openweathermap.org).
+
+Requires an API key for OpenWeatherMap (OWM), but the free tier allows you
+enough requests/sec to get accurate weather even up to the second.
+
+Note that this module does require an additional python package,
+pyowm (https://github.com/csparpa/pyowm), and this may be easily
+installed via pip.
+
+This module allows you to specify an icon for nearly every weather scenario
+imaginable. The default configuration options lump many of the icons into
+a few groups, and due to the limitations of UTF-8, this is really as expressive
+as it gets.
+
+I would highly suggest you install an additional font, such as the incredible
+(and free!) Weather Icons font (https://erikflowers.github.io/weather-icons),
+which has icons for most weather scenarios.
+
+For more information, see the documentation
+(ttps://openweathermap.org/weather-conditions) on what weather conditions are
+supported. See the configuration options for how to specify each weather icon.
+
+Configuration parameters:
+    api_key: Your OpenWeatherMap API key.
```

Comment:
Opinion: Required prefix? `api_key: Required. Your OpenWeatherMap API key.`

@tobes This is new. We standardize things. Ought to decide if we
want `api_key` (this), `token` (aqicn), or `auth_token` (github) soon before we get
more API modules. The next one may be `key` or `key_auth`.